src/go/configmanager/config_manager_test.go

```diff
@@ -46,6 +46,7 @@ import (
 	wrapperspb "github.com/golang/protobuf/ptypes/wrappers"
 	annotationspb "google.golang.org/genproto/googleapis/api/annotations"
 	confpb "google.golang.org/genproto/googleapis/api/serviceconfig"
+	googleapiscpb "google.golang.org/genproto/googleapis/api/servicecontrol/v1"
```

Comment:
`scpb`? to match the other imports that are also googleapis?

Reply:
Please fix this across all files, not only this one

Reply:
I use this because scpb has been occupied by `scpb "github.com/GoogleCloudPlatform/esp-v2/src/go/proto/api/envoy/http/service_control"` in L37
Change to servicecontrolpb; hope it can make you feel better.